src/nat/cli/commands/red_teaming/red_teaming_utils.py

```
@@ -59,23 +58,16 @@ def load_red_teaming_config(config_file: Path) -> RedTeamingRunnerConfig:
-            raise ValueError(
-                f"Unsupported file format: {config_file.suffix}. "
-                "Use .yml, .yaml, or .json"
-            )
+            raise ValueError(f"Unsupported file format: {config_file.suffix}. "
+                             "Use .yml, .yaml, or .json")
 
     if not isinstance(config_data, dict):
-        raise ValueError(
-            f"Red teaming config file must contain a dictionary, got {type(config_data)}"
-        )
+        raise ValueError(f"Red teaming config file must contain a dictionary, got {type(config_data)}")
```

Ruff (0.14.8)

Prefer `TypeError` exception for invalid type

(TRY004)